server/models/annotation.js

```diff
@@ -17,10 +17,10 @@ const rangeSchema = new Schema({
 }, { _id: false });
 
 const annotationSchema = new Schema({
-  author: { type: ObjectId, ref: 'User' },
+  author: { type: ObjectId, ref: 'User', autopopulate: { select: '-_id' } },
```

Comment:
Fun fact: this is necessary so that the total # of annotations on an article can still be found by counting the _id's